bitnami/external-dns/templates/clusterrole.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.rbac.create }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I see that the official helm chart is parametrizing the apiVersion (https://github.com/helm/charts/blob/master/stable/external-dns/templates/clusterrole.yaml#L2)

Do you think is a good feature to add to this chart? I would say yes